langchain4j-core/src/main/java/dev/langchain4j/data/image/Image.java

```diff
@@ -65,6 +69,19 @@ public String toString() {
                 " }";
     }
 
+    public static Image fromPath(Path path) {
```

Comment:
OK. I hadn't looked at `ImageContent` and how it was working.
My `fromPath()` method was only used in my test, so I'll just move it in my test class as a local utility method for my test.

I wonder 2 things though:
- shouldn't `Image` also have a mime type field as well?
- could `ImageContent` also have `from()` methods that take `Path`, so that it's easier to load an image from the local file system?

Reply:
Oops, sorry, yes, at some point there was no `mimeType` when I had started working on that PR, but it's here now!

Now, I'm good with the changes.
Please, when you have a moment, double check the latest changes 🙏